cli/src/main/package/bin/ide.bat

```diff
@@ -5,6 +5,20 @@ Set _fBGreen=[92m
 Set _fBRed=[91m
 Set _RESET=[0m
 
+rem Add Git PATH entries for CMD - https://github.com/devonfw/IDEasy/issues/764
+for %%H in ( HKEY_LOCAL_MACHINE HKEY_CURRENT_USER ) do for /F "usebackq tokens=2*" %%O in (`call "%SystemRoot%"\system32\reg.exe query "%%H\Software\GitForWindows" /v "InstallPath" 2^>nul ^| "%SystemRoot%\system32\findstr.exe" REG_SZ`) do set GIT_HOME=%%P
+
+set "GIT_BIN=%GIT_HOME%\usr\bin"
+set "GIT_CORE=%GIT_HOME%\mingw64\libexec\git-core"
+
+if exist "%GIT_BIN%" (
+	echo "%PATH%" | find /i "%GIT_BIN%">nul  || set PATH=%PATH%;%GIT_BIN%
```

Comment:
This is not working. Please test your code before providing your solution for review. When I merge and we release without further testing end-users will get blocked.
```suggestion
	echo "%PATH%" | find /i "%GIT_BIN%">nul  || set "PATH=%PATH%;%GIT_BIN%"
```